
This patch corrects "aplay: set_params:1012: unable to install sw params:" bug.

--- buildroot/package/audio/alsa-lib/alsa-lib.mk.org	2008-12-11 14:35:36.000000000 +0100
+++ buildroot/package/audio/alsa-lib/alsa-lib.mk	2008-12-11 14:43:07.000000000 +0100
@@ -45,6 +45,7 @@
 		--enable-shared \
 		--enable-static \
 		--disable-docs \
+		--with-versioned=no \
 		$(SOFT_FLOAT) \
 		$(DISABLE_NLS) \
 	)
